framework/src/utils/InputParameters.C

```diff
@@ -277,6 +279,20 @@ InputParameters::isPrivate(const std::string &name) const
 }
 
 void
+InputParameters::declareControllable(const std::string & input_names)
+{
+  std::vector<std::string> names;
+  MooseUtils::tokenize<std::string>(input_names, names, 1, " ");
```

Comment:
Just a comment: Space delimited strings can be split naturally with normal istream_iterators with no external dependencies or logic (i.e. a one liner). This is fine though.